cmd/root.go

```diff
@@ -44,6 +50,21 @@ var rootCmd = &cobra.Command{
 	// Uncomment the following line if your bare application
 	// has an action associated with it:
 	// Run: func(cmd *cobra.Command, args []string) { },
+	PersistentPreRun: func(cmd *cobra.Command, args []string) {
```

Comment:
We are not printing any error if Github is down https://github.com/yugabyte/ybm-cli/pull/107/files#diff-ab967ab1a2f3a1b769106eeb7bfe892ef0e81d1d27811fa15be08e6749feee1fR57

Reply:
Also, since printing the string is not causing any harm, even with jq, I am not sure how adding the additional env var would help? I am happy to make the change if needed though.